include/dxc/Test/HlslTestUtils.h

```diff
@@ -336,8 +371,9 @@ inline bool GetTestParamBool(LPCWSTR name) {
   if (NameValue.IsEmpty()) {
     return false;
   }
-  return Unicode::IsStarMatchWide(ParamValue, ParamValue.GetLength(),
-                                  NameValue, NameValue.GetLength());
+
+  return hlsl_test::IsStarMatchWide(ParamValue, ParamValue.GetLength(),
```

Comment:
Why cannot use Unicode::IsStarMatchWide here?

Reply:
Because we don't want to depend on dxcSupport?

Reply:
Yes, we want to avoid linking DXC support lib to the ExecutionTests and therefore the HLK tests.